EventAggregator.Tests/SpecExtensions/xUnitSpecificationExtensions.cs

```diff
@@ -1,306 +1,290 @@
-    {
-        Assert.DoesNotThrow(method);
-    }
-
-
-
-    public static IEnumerable<T> Single<T>(this IEnumerable<T> collection)
-    {
-        Assert.Single<T>(collection);
-        return collection;
-    }
-
-
-
-    public static T IsInRange<T>(this T actual, T low, T high) where T : IComparable
-    {
-        Assert.InRange<T>(actual, low, high);
-        return actual;
-    }
-
-    public static T IsInRange<T>(this T actual, T low, T high, IComparer<T> comparer)
-    {
-        Assert.InRange<T>(actual, low, high, comparer);
-        return actual;
-    }
-
-
-
-    public static T IsNotInRange<T>(this T actual, T low, T high) where T : IComparable
-    {
-        Assert.NotInRange<T>(actual, low, high);
-        return actual;
-    }
-
-    public static T IsNotInRange<T>(this T actual, T low, T high, IComparer<T> comparer)
-    {
-        Assert.NotInRange<T>(actual, low, high, comparer);
-        return actual;
-    }
-
-    public static IEnumerable IsEmpty(this IEnumerable collection)
-    {
-        Assert.Empty(collection);
-        return collection;
-    }
-
-    public static T IsType<T>(this T actual, Type expectedType)
-    {
-        Assert.IsType(expectedType, actual);
-        return actual;
-    }
-
-    public static T IsNotType<T>(this T actual, Type expectedType)
-    {
-        Assert.IsNotType(expectedType, actual);
-        return actual;
-    }
-
-    public static T IsAssignableFrom<T>(this T actual, Type expectedType)
-    {
-        Assert.IsAssignableFrom(expectedType, actual);
-        return actual;
-    }
-
-
-/* Consider implementing these later 
- 
-    public static string ShouldBeEqualIgnoringCase(this string actual, string expected)
-    {
-        StringAssert.AreEqualIgnoringCase(expected, actual);
-        return actual;
-    }
-
-    public static string ShouldStartWith(this string actual, string expected)
-    {
-        StringAssert.StartsWith(expected, actual);
-        return actual;
-    }
-
-    public static string ShouldEndWith(this string actual, string expected)
-    {
-        StringAssert.EndsWith(expected, actual);
-        return actual;
-    }
-
-    public static void ShouldBeSurroundedWith(this string actual, string expectedStartDelimiter, string expectedEndDelimiter)
-    {
-        StringAssert.StartsWith(expectedStartDelimiter, actual);
-        StringAssert.EndsWith(expectedEndDelimiter, actual);
-    }
-
-    public static void ShouldBeSurroundedWith(this string actual, string expectedDelimiter)
-    {
-        StringAssert.StartsWith(expectedDelimiter, actual);
-        StringAssert.EndsWith(expectedDelimiter, actual);
-    }
-
-    public static void ShouldContainErrorMessage(this Exception exception, string expected)
-    {
-        StringAssert.Contains(expected, exception.Message);
-    }
-*/
-}
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Xunit;
+
+
+public static class xUnitSpecificationExtensions
```

Comment:
Not to do with this PR, but give https://github.com/shouldly/shouldly a try. It has all these extension methods and gives *way* better error messages than other assertion frameworks